### ISSUE TITLE
Multilingual models for 2 and more languages

### ISSUE THREAD

**Issue by ifyousleep** [alphacep/vosk-api#233]
Good afternoon
Is it realistic to combine two models for an android application into one common - Russian and English?
I would be grateful for your help

**Comment by nshmyrev** [comment]
> Is it realistic to combine two models for an android application into one common - Russian and English?

Yes, sure, it is not a big problem. You have to train yourself though.

**Comment by ifyousleep** [comment]
thanks
can you direct me towards instructions on how to do this, if an instruction exists?

**Comment by nshmyrev** [comment]
I do not think such instruction exists, you simply train a model for both languages treating it as a single language.

**Comment by ifyousleep** [comment]
Do you mean this paragraph?
"You can train your model with Kaldi toolkit. The training is pretty standard - you need tdnn nnet3 model with ivectors. You can check mini_librispeech recipe for details."

Thanks

**Comment by nshmyrev** [comment]
Good language detector:

https://huggingface.co/TalTechNLP/voxlingua107-epaca-tdnn

**Comment by Pramodk-dev** [comment]
please provide m model training recipe.
thanks
